README.md

```diff
@@ -131,6 +131,8 @@ client.shutdown()
 
 NOTE: Parallel IO with Dask is enabled once a valid chunk size is passed to `load` routine using `chunks` parameter. For `save` routine, the DaskIO is enabled only if the array passed is a Dask Array. Currently only `numpy` format supports this functionality.
 
+### IO with HDF5
+
```

Comment:
Could you write also few details here?

Reply:
Let me know if I need to explain anything else in more detail.

Reply:
It's good, thanks!